crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -378,6 +378,7 @@ impl SlidingSync {
         Ok(update_summary)
     }
 
+    #[instrument(skip_all, fields(pos))]
```

Comment:
Hmm, AFAIR I avoided adding an instrument here because it looked weird in Jaeger when used in combination with the stream. I guess trying to record the pos in the parent span is even weirder.

Reply:
I think the key is that `sync_all` needs to be called with `.instrument(parent_span)` in the `stream! {}` still, since it won't otherwise have a parent at all. I think we should still have a proper tree after this PR.